_rules/6cbcbe.md

```diff
@@ -0,0 +1,279 @@
+Resolving the links includes potential redirects, if the redirects happen instantly.
+
+## Assumptions
+
+This rule assumes that the purpose of the links with identical names would not be ambiguous to users in general when seen in context on the web page, as users of assistive technologies in this case is not at a disadvantage when viewing the link out of context, e.g. on a list of links in a screen reader.
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [Understanding Success Criterion 2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)
+- [URL parsing](https://www.w3.org/TR/html52/infrastructure.html#parsing-urls)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+A set of two HTML `<a>` elements have the same accessible name and link to the same resource.
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+```
+
+#### Passed example 2
+
+Links resolves to same resource after instant redirect:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/redirect.html">Contact us</a>
+```
+
+#### Passed example 3
+
+Resources are not the same, since the links resolve to different URLs, but the resources are completely identical, thus serving the same purpose:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/index-copy.html">Contact us</a>
+```
+
+#### Passed example 4
+
+Same link text used for links going to pages where the content section is the same, but where the navigation options (bread crumbs and local sub menus) differ due to different placement in navigation hierarchy:
+
+```html
+<a href="/test-assets/6cbcbe/about/contact.html">Contact us</a>
+<a href="/test-assets/6cbcbe/careers/contact.html">Contact us</a>
+```
+
+#### Passed example 5
+
+URLs differ due to trailing slashes, but resolves to the same resource after redirects caused by user agent:
+
+```html
+<a href="/test-assets/6cbcbe/">Contact us</a> 
+<a href="/test-assets/6cbcbe">Contact us</a>
+```
+
+#### Passed example 6
+
+Pages contain different amounts of information and/or differently worded information, but fulfils the same purpose in relation to the link:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html">Call us</a>
+<a href="/test-assets/6cbcbe/page2.html">Call us</a>
+```
+
+#### Passed example 7
+
+Pages have the same advertised key content but use different layouts:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html">Contact us</a>
+<a href="/test-assets/6cbcbe/page3.html">Contact us</a>
+```
+
+#### Passed example 8
+
+Links created via scripting with explicit role of link, but leads to the same resource: 
+
+```html
+<span role="link"
```

Comment:
I know that is not the point, but can we just add `tabindex="0"` to every  `span` with `role="link"` :-)

Otherwise, the test case won't be keyboard usable (and since it's not the point of the test, I think it's a little damaging)